tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml.cs

```diff
@@ -167,21 +171,26 @@ private StackPanel BuildNavItem(BitmapImage image, string text)
             {
                 Width = 16,
                 Height = 16,
-                Margin = new Thickness(0, 0, 10, 0),
+                Margin = new Thickness(0, 0, 9, 0),

```

Comment:
Should this actually be 8? Usually we want multiples of 4 (which 10 wasn't anyway)

Reply:
Done!